test/e2e/sriovoperatornodepolicy_test.go

```diff
@@ -32,7 +32,7 @@ var _ = Describe("Operator", func() {
 		Eventually(func() *cluster.EnabledNodes {
 			sriovInfos, _ = cluster.DiscoverSriov(clients, testNamespace)
 			return sriovInfos
-		}, timeout, interval).ShouldNot(BeNil())
+		}, time.Minute*5, time.Second*20).ShouldNot(BeNil())
```

Comment:
can you use multiples of values defined at the [test suite](https://github.com/rollandf/sriov-network-operator/blob/nodestatepolicy/test/e2e/e2e_tests_suite_test.go#L42-L44) instead of hard coding it?

Reply:
Done